src/Action_GIST.cpp

```diff
@@ -393,13 +393,13 @@ Action::RetType Action_GIST::Setup(ActionSetup& setup) {
       isFirstSolvent = false;
     } else {
       // This is a non-solvent molecule. Save atom indices unless 1 atom (probably ion).
-      if (mol->NumAtoms() > 1) {
+      //if (mol->NumAtoms() > 1) {
```

Comment:
Maybe instead of turning it off outright, we can add a keyword called `includeions` or something similar that will have the same effect, e.g.
```C++
if (mol->NumAtoms() > 1 || includeIons_) {
```
this will preserve backwards compatibility while making the routine more flexible.

Reply:
Thanks Dan, that looks great.  I (Tom here) would like includeIons to default to true as I think that most users will just run the defaults and not be aware that relevant ions that are effectively part of the protein will be ignored if the default is false.

Reply:
>  I (Tom here) would like includeIons to default to true as I think that most users will just run the defaults and not be aware that relevant ions that are effectively part of the protein will be ignored if the default is false.

Sounds fine. When the change is made can you also increment the minor version number (second number) in `Version.h` since this will alter results from what was previously output from cpptraj GIST? Thanks!